cdk/bin/main.ts

```diff
@@ -19,6 +19,7 @@ const stackPrefix = "PerformanceDash-".concat(envName);
 const accountId = cdk.Aws.ACCOUNT_ID;
 const region = cdk.Aws.REGION;
 const datasetsBucketName = `performancedash-${envName.toLowerCase()}-${accountId}-${region}-datasets`;
+const contentBucketName = `performancedash-${envName.toLowerCase()}-${accountId}-${region}-customcontent`;
```

Comment:
Bucket names can be up to 63 characters long. Should we name the bucket just `content` instead of `customcontent` to shorten it by 6 characters and leave more room to the Environment name? I'm afraid that if a customer enters a long Environment Name, the bucket will exceed the 63 chars and fail the deployment. 

What do you think?

Reply:
Good call. I also name it content everywhere else. So I think that makes sense.